addons/edi/models/edi_gateway.py

```diff
@@ -193,8 +193,17 @@ def _compute_path_count(self):
     @api.depends('transfer_ids')
     def _compute_transfer_count(self):
         """Compute number of transfers (for UI display)"""
+
+        # We use the read_group method to aggregate data from the edi.transfer model. 
```

Comment:
Nit: Better to update or add your comment in the doc string, instead of having a docsting and a comment just after docstring

Reply:
changed